### DISCUSSION TITLE
How to skip or reset Dedup Interval?

### DISCUSSION THREAD

**Discussion by adrianulima** [vercel/swr#1623] [Q&A]
Hi 👋 

I have a response interceptor that redirects user back to login if the token is invalid, but if the user login again before `dedupingInverval` ends my page's content is not loaded.
I'm using a high value for `dedupingInterval` like 5000 ms to prevent multiple repeated calls while navigating between pages. 

So the first api call fails because of the token, then when the user gets back to the page it is not fetching data again (even the first one failing).

I tried calling `cache.clear()` but dedup info is no on cache or mutate, is there a way clear all concurrent_promises? I saw there is a new `unmount` on `initCache` response, is there a way to access it?

**Comment by muqg** [comment]
I was about to ask a very similar question. I have a use case where I need to wipe SWR back to zero. `cache.clear` was my first intention and it works great... except for the `useSWR` calls which are still within their dedupe interval. None of the latter will be cleared, and they will keep on showing their last cached data.

For the purpose of science I tried a `console.log(cache)` just before calling `cache.clear` and another `console.log(cache)` right after clearing the cache. Seems that the `useSWR` calls with a _high dedupe interval_ just insert their last data (data they had before the cache clear) back into the cleared cache, disregarding the fact that their data should now be empty and are thus expected to _revalidate_ or something.

**Comment by pz129** [comment]
Is there any update on this issue? seems like a common case
